schemas/resources/schemas/data-quality-summary/v1.0.0/search.md

```diff
@@ -26,7 +26,7 @@ These parameters will match fields within a Data Quality Summary. They are case
 * provider - options: pattern, ignore\_case
 * native\_id - options: pattern, ignore\_case
 * concept\_id
-* id (legacy identifier)
+* id
```

Comment:
Please undo this change.  The id is the legacy services identifier. It isn't a random id.

Reply:
They specifically change it to just id.

Reply:
in all the files.